tests/service/ProjectAnalysisTests.fs

```diff
@@ -5195,3 +5195,16 @@ type A(i:int) =
 
     | Some decl -> failwithf "unexpected declaration %A" decl
     | None -> failwith "declaration list is empty"
+
+
+[<TestCase(([||]: string[]), ([||]: bool[]))>]
```

Comment:
This isn't urgent, but just to say that I personally never use ``TestCase`` attributes in the F# tests, and prefer a loop over an F# list of test case data inputs.  Indeed I usually remove any uses of ``TestCase`` when I come across them later

While using TestCase attribute has some advantages (e.g. they get reported as separate tests in a test explorer), I believe that just using F# code to program the space of test cases is more uniform and in the long run easier to understand - and encourages people to be really expansive in how they meta-program up a very broad range of systematic test cases.

Reply:
(Programming up the test case matrix with F# data is also more typesafe of course)